src/index.html

```diff
@@ -13,6 +13,57 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--yellow">
```

Comment:
Use semantic tags where possible. For example, you could wrap the .stars element in a <section> or <div> tag with a semantic class name.